frontend/ui/widget/configdialog.lua

```diff
@@ -927,8 +927,7 @@ function ConfigDialog:onShowConfigPanel(index)
 end
 
 function ConfigDialog:onConfigChoice(option_name, option_value)
-    self.configurable[option_name] = option_value
-    self.ui:handleEvent(Event:new("StartActivityIndicator"))
+    self.ui:handleEvent(Event:new("ConfigChange", option_name, option_value))
```

Comment:
There's no onConfigChange with cre/ReaderRolling, so the option might not be saved in self.configurable ?

Reply:
I forgot to add it, will do soon, Thanks

Reply:
done